dashboard/app/models/pd/workshop.rb

```diff
@@ -90,6 +90,29 @@ class Pd::Workshop < ActiveRecord::Base
   }.freeze
   SECTION_TYPES = SECTION_TYPE_MAP.values.freeze
 
+  # Time constrains for payment, per subject.
+  # Each subject has the following constraints:
+  # min_days: the minimum # of days a teacher must attend in order to be counted at all.
+  # max_days: the maximum # of days the workshop can be recognized for.
+  # max_hours: the maximum # of hours the workshop can be recognized for.
+  TIME_CONSTRAINTS_BY_SUBJECT = {
```

Comment:
This name is a little weird - it seems to be more than just the time constraints on the workshop